sample-dcrdata.conf

```diff
@@ -94,3 +94,6 @@
 ; Maximum number of comma-separated addresses allowed in certain Insight API
 ; endpoints, such as /insight/api/addrs/{addr0,..,addrN}
 ;max-api-addrs=3
+
+; TOR hidden service address.  When specified, it will be displayed in the footer.
+;onionaddress=
```

Comment:
This needs to be "onion-address" instead of "onionaddress" to match the config struct tags.  Also, please add a trailing newline at the end of the file.

Reply:
Thanks for spotting, fixed.